### ISSUE TITLE
ResourceManager relative url check fails

### ISSUE THREAD

**Issue by eliottrobson** [AndiDittrich/WordPress.Enlighter#61]
The ResourceManager getResourceUrl regex fails:
preg_match('#^(?:/|[a-z]:/).*$#i', $filename) === 0

when given an absolute URL (custom theme).

What is this supposed to do? Maybe a replacement such as:

preg_match('/^(https?)?\:\/\//', $filename)

could work?

**Comment by AndiDittrich** [comment]
Dear Eliott,

thanks for your report!
I can confirm that external themes (`enlighter/` subdirectory in your theme) are not correctly loaded.

The ResourceManager is responsible for the invalid URL but it is not the issue :)
In ResourceLoader.php, the external Themes are enqueued the wrong way - i am sorry for that - i've forgott to update the code!

```php
// load user themes ?
if ($this->_config['embedExternalThemes']) {
    // embed available external themes
    foreach ($this->_themeManager->getUserThemes() as $theme => $sources) {
        $this->enqueueStyle('enlighter-external-' . strtolower($theme), $sources[1]);
    }
}
```

I will fix it asap!

best regards, Andi

**Comment by AndiDittrich** [comment]
And of course..the regex seems to be weak...sry!

**Comment by AndiDittrich** [comment]
Fixed in v3.4-BETA5 -  available within the current master branch

Well .. .a simple ``+`` (plus) sign was missing in the regex. I've chosen these "strange" version to match external URLs with protocols like `http://` as well as without protocols `//fonts.google.com`

**Comment by eliottrobson** [comment]
Excellent, I'll update to the beta version and look forwards to the wp repository update.
